tutorials/packaging/create-your-first-snap.md

```diff
@@ -171,336 +192,416 @@ grade: devel
 confinement: devmode
 ```
 
-
 positive
-: A quick note on the **version** bit: it’s put into quotation marks ('`2.10`') here, because the version is a string, not a number. You could theoretically use a version string without any numbers (like '`myfirstversion`') and that would be fine too. This version is only for user consumption and doesn’t require anything like ascii ordering (e.g ver1 > ver2 and the like) for an update to reach the user.
-```
-
-Simple, isn’t it?
+: The version is quoted ('`2.10`') because the version is a string, not a number. You could
+theoretically use a version string without numbers (like '`myfirstversion`'). This information is
+for user consumption only and doesn't require special ordering (e.g ver1 > ver2) for an update to
+reach the user.
 
 ### Adding a part
 
 A snap can consist of multiple parts. Here are a few examples of this:
 
-  - Snaps with separate logical parts, e.g. a server snap, which contains a web server, a database and the application itself or a game which ships the game engine and game data for three different games, each one being defined in its own part.
+  - Snaps with separate logical parts, e.g. a server snap, which contains a web server, a database
+    and the application itself or a game which ships the game engine and game data for three
+    different games, each one being defined in its own part.
   - Snaps with parts which come from different locations.
   - Parts which are built in a different way.
 
-Our **hello** snap will be nice and simple and just consist of one part for now. In the following pages we are going to extend it step by step.
-
-Two must-haves for every part are the **source** and **plugin** definition. Think of it as the what (source) and how (plugin). As source you can for example pick a source repository (like **git**), a **tarball**, or a **local directory**. Snapcraft comes with a lot of plugins which can build almost any conceivable project type (e.g. autotools, cmake, go, maven, nodejs, python2, python3, etc).
+Our **hello** snap will be nice and simple. It will consist of only one part for now. In the
+following pages we are going to gradually extend it.
 
-To build **hello**, simply add the following to your `snapcraft.yaml` file:
+Two must-haves for every part are the **source** and **plugin** definition. Think of it as the
+"what" (source) and the "how" (plugin). As source you can, for example, pick a source repository
+(like **git**), a **tarball**, or a **local directory**. Snapcraft comes with a lot of plugins
+which can build almost any conceivable project type (e.g. autotools, cmake, go, maven, nodejs,
+python2, python3, etc).
 
+To build **hello**, add the following to your `snapcraft.yaml` file:
 
-```bash
+```yaml
 parts:
   gnu-hello:
 	source: http://ftp.gnu.org/gnu/hello/hello-2.10.tar.gz
 	plugin: autotools
 ```
 
```

Comment:
the on screen version has the wrong indent (not valid yaml) probably replace these tabs(?)

Reply:
Most of these YAML excerpts are like that. I don't know why this one stands out. Anyway, I went through them all (using 2-space indenting).

Also, only the initial PR gets a demo built.